proxy/healthy_endpoints.go

```diff
@@ -22,6 +23,7 @@ func (h *healthyEndpoints) filterHealthyEndpoints(endpoints []routing.LBEndpoint
 	for _, e := range endpoints {
 		if p < e.Metrics.HealthCheckDropProbability() {
 			/* drop */
+			metrics.IncCounter("routing.endpoint." + e.Host + ".optout.loadbalancer")
```

Comment:
I think it makes sense to store not only amount of requests being dropped from particular endpoint (as you do in this line) but also to count amount of requests which have at least one endpoint being dropped. It could be done after all iterations of this loop.